<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KaGuiMainWindow</class>
 <widget class="QMainWindow" name="KaGuiMainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ka GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="margin">
     <number>4</number>
    </property>
    <item>
     <widget class="QGroupBox" name="xmitterBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Transmitter</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="xmitterStatusLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#8b0000;&quot;&gt;waiting for status...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="xmitterControlAndStateFrame">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Control and State</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="margin">
           <number>6</number>
          </property>
          <item row="2" column="1">
           <widget class="QLabel" name="xmitterWarmupLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Filament Warmup</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="xmitterCooldownIcon">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="KaGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="xmitterWarmupIcon">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="KaGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="xmitterStandbyIcon">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="KaGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="xmitterHvpsRunupIcon">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="KaGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QFrame" name="faultFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="xmitterFaultLabel">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Fault</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="xmitterFaultDetailsButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>24</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="xmitterHvpsRunupLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>HVPS Runup</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="xmitterCooldownLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Cooldown</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="xmitterFaultIcon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="KaGuiMainWindow.qrc">:/redLED_off.png</pixmap>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="xmitterStandbyLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Standby</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="xmitterHvpsOnLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>HVPS On</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="xmitterHvpsOnIcon">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="KaGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QFrame" name="xmitterControlFrame">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="xmitterPowerButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="KaGuiMainWindow.qrc">
                 <normaloff>:/powerbutton.png</normaloff>:/powerbutton.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="xmitterStandbyButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="text">
                <string>Standby</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="xmitterOperateButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="text">
                <string>Operate</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="xmitterUnitOnIcon">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="KaGuiMainWindow.qrc">:/greenLED_off.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="xmitterValuesFrame">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Monitor</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="verticalSpacing">
           <number>4</number>
          </property>
          <item row="2" column="2">
           <widget class="QLabel" name="xmitterHvpsCurrentUnits">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>mA</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="xmitterTemperatureValue">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>00</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="xmitterTemperatureUnits">
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="xmitterHvpsVoltageLabel">
            <property name="text">
             <string>HVPS Voltage </string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="xmitterHvpsVoltageUnits">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>kV</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="xmitterHvpsVoltageValue">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>00.0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="xmitterHvpsCurrentLabel">
            <property name="text">
             <string>HVPS Current</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="xmitterHvpsCurrentValue">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>00.0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="xmitterMagCurrentUnits">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>mA</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="xmitterTemperatureLabel">
            <property name="text">
             <string>Temperature</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="xmitterMagCurrentLabel">
            <property name="text">
             <string>Magnetron Avg. Current </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="xmitterMagCurrentValue">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>00.0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="kadrxBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>kadrx</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="kadrxStatusLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#8b0000;&quot;&gt;waiting for status...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="kadrxXmitDisableFrame">
         <property name="title">
          <string>Transmit Disable</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="4" column="0">
           <widget class="QLabel" name="inBlankingSectorIcon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="KaGuiMainWindow.qrc">:/redLED_off.png</pixmap>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="n2WgPresLabel">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;N&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt; Waveguide Pressure Low&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="disabledByRequestLabel">
            <property name="text">
             <string>Disabled via XML-RPC</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="disabledByRequestIcon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="KaGuiMainWindow.qrc">:/redLED_off.png</pixmap>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="n2WgPresIcon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="KaGuiMainWindow.qrc">:/redLED_off.png</pixmap>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="inBlankingSectorLabel">
            <property name="text">
             <string>Blanking Sector</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="noPentekTriggersLabel">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>No Triggers From Pentek</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="noPentekTriggersIcon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="KaGuiMainWindow.qrc">:/redLED_off.png</pixmap>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
      <zorder>verticalSpacer</zorder>
      <zorder>kadrxStatusLabel</zorder>
      <zorder>kadrxXmitDisableFrame</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="KaGuiMainWindow.qrc"/>
  <include location="KaGuiMainWindow.qrc"/>
  <include location="../../../../../../h/eol/burghart/git/Ka-band_instrument/src/ka_gui/KaGuiMainWindow.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>powerClicked()</slot>
  <slot>faultResetClicked()</slot>
  <slot>standbyClicked()</slot>
  <slot>operateClicked()</slot>
 </slots>
</ui>
